Assets/VRTK/SDK/Daydream/DaydreamControllerLog.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
```

Comment:
i dont think so, i was using it to explore the grv sdk, and then just left it in. ok? or i can remove